Backend/Remora.Discord.Rest/API/RateLimitBucket.cs

```diff
@@ -112,7 +113,7 @@ public static bool TryParse(HttpResponseHeaders headers, [NotNullWhen(true)] out
                 return false;
             }
 
-            if (!double.TryParse(rawReset.SingleOrDefault(), out var resetsAtEpoch))
+            if (!double.TryParse(rawReset.SingleOrDefault(), NumberStyles.Float | NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture, out var resetsAtEpoch))
```

Comment:
Use `NumberStyles.Float | NumberStyles.AllowThousands` instead, as that's the default flag set.